packages/wrangler/package.json

```diff
@@ -94,7 +94,11 @@
 		},
 		"transformIgnorePatterns": [
 			"node_modules/.pnpm/(?!find-up|locate-path|p-locate|p-limit|p-timeout|p-queue|yocto-queue|path-exists|execa|strip-final-newline|npm-run-path|path-key|onetime|mimic-fn|human-signals|is-stream|get-port|supports-color|pretty-bytes|npx-import)"
-		]
+		],
+		"snapshotFormat": {
+			"escapeString": true,
+			"printBasicPrototype": true
+		}
```

Comment:
Should this cause the snapshots to need updating?

Reply:
This prevents the snapshots from needing updating with the v29 migration
https://jestjs.io/docs/upgrading-to-jest29#snapshot-format